funsor/tensor.py

```diff
@@ -924,7 +920,7 @@ def numeric_tensordot(x, y, dims):
     return Einsum(equation, (x, y))
 
 
-def numeric_stack(parts, dim=0):
+def stack(parts, dim=0):
```

Comment:
Interesting, I think we could later replace this with an eager evaluation rule for `ops.stack` and eliminate `funsor.tensor.stack` entirely. Similarly I think we'll be able to replace `tensordot` with an `ops.tensordot` + an `eager` rule. But that's out of scope for this PR :smile: